src/net/sparkzz/modest/io/config/YAMLConfig.java

```diff
@@ -84,9 +84,7 @@ else if (((Number) tempObject).intValue() == 0)
 	public boolean hasValue(String key) {
 		tempObject = data.get(key);
 
-		if (tempObject != null)
-			return true;
-		return false;
+		return (tempObject != null);
```

Comment:
Much more convenient and easy-to-read when you simplify return statements like this.